types/office-js-preview/index.d.ts

```diff
@@ -5204,6 +5230,156 @@ declare namespace Office {
          */
         getPrefixAsync(ns: string, callback?: (result: AsyncResult<string>) => void): void;
     }
+    /**
+     * Provides methods for an add-in to request permission from a user to access their device capabilities.
+     * A user's device capabilities include their camera, geolocation, and microphone.
+     *
+     * @remarks
+     *
+     * **Applications**: This API is supported by the following Office applications when running in Chromium-based browsers,
+     * such as Microsoft Edge and Google Chrome.
+     *
+     * - Excel on the web
+     *
+     * - Outlook on the web
+     *
+     * - PowerPoint on the web
+     *
+     * - Word on the web
+     *
+     * **Requirement set**: {@link https://learn.microsoft.com/javascript/api/requirement-sets/common/device-permission-service-requirement-sets | DevicePermission 1.1}
+     */
+    interface DevicePermission {
+        /**
+         * Requests permission from a user to access their device capabilities, such as a camera or microphone.
+         *
+         * All the requested permissions are displayed in a single modal dialog to the user.
+         * The dialog includes options to **Allow**, **Allow once**, or **Deny** the requested permissions.
+         *
+         * This method returns a promise and is supported in web versions of Excel, PowerPoint, and Word running in a Chromium-based browser, such as Microsoft Edge or Google Chrome.
```

Comment:
I think you can let the interface description do the heavy-lifting and avoid repeating all the details here. That way, the focus can be on this being for WXP.
```suggestion
         * This method returns a promise. Use it with Excel, PowerPoint, and Word add-ins.
```